plugins/org.jboss.tools.locus.jsonschema2pojo/0.4.13/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,9 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: org.jboss.tools.locus.jsonschema2pojo.jsonschema2pojo-core
+Bundle-Version: 0.4.13
+Bundle-ClassPath: .
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Bundle-RequiredExecutionEnvironment: JavaSE-1.6

```

Comment:
Hold off on the PR for a bit - I'll rebuild Data Transformation with an updated JBTIS TP referencing my local copy of locus (updated for exports).  thkx